spec/support/tests/current_bundle.rb

```diff
@@ -32,12 +40,14 @@ def possible_appraisals
       appraisals
     end
 
-    def path
-      Bundler.default_gemfile
+    def current_appraisal
+      if appraisal_in_use?
+        File.basename(path, ".gemfile")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.